### ISSUE TITLE
create events and show events

### ISSUE THREAD

**Issue by stefan-dsr** [craigk5n/webcalendar#333]
My "old" calendar in version 1.30 works fine an I wanted to change to version 1.91. After installing a new calendar in version 1.9.1 I miss the "+" button to create a new event on a date. When I double-click a specified day in calendar the menue for creating an event is opening and I can fill the necessary fields in the mask.
After "Saving" the event, I can't see this event in calendar. When searching the event in the database I have always events in November 2nd 2021 (but not visible in calendar). I createt my Test-events as administrator. 
Are there any solutions for this fault?

**Comment by GemeinsamGemeinde** [comment]
Have you activate the layers? Only the activated layers will shown in the public calender view...

**Comment by stefan-dsr** [comment]
Yes, I activated layers. My calendar vers. 1.9.1 has the same settings as the calendar vers. 1.2.9, but the new version doesn't work correct. It seems the calendar has also problems with the date.

 

Here the facts of one event I created:

 

Beschreibung              Test

Datum                        Mi, 22.20.0 (fault, correct entry: wednesday 2022-12-28)

Startzeit                      01:00-02:00 CET (fault, correct: 16:00-17:00)

Priorität                       5-Normal

Zugang                       Öffentlich(e)

Erstellt von                 xxxxxxx          

Aktualisiert                 Di, 2.11.2021 01:00 CET (fault, correct: thuesday 2022-12-27)

 

regards Stefan

 

Von: GemeinsamGemeinde ***@***.*** 
Gesendet: Dienstag, 27. Dezember 2022 17:17
An: craigk5n/webcalendar
Cc: stefan-dsr; Author
Betreff: Re: [craigk5n/webcalendar] create events and show events (Issue #333)

 

Have you activate the layers? Only the activated layers will shown in the public calender view...

**Comment by GemeinsamGemeinde** [comment]
:-) First, please don't answer on the notification mail from github, this will placed in the forum ....
When it is possible for you to add an new database inside of your hosting contract, try to create an new database, run an new installation of webcalendar maybe in an created subdomain and when it is configured ready, you can link it back to your default url
If you have a limit at your hosting contract look at the database functionalities where you can export the settings from your existing database to an dumpfile (e.g. at https://mein.ionos.de  its quite simple and guided, also at all-inkl.com), that is the standard recomended backup due the update, but don't make the update, Install new and import after (when nessesarry the restly events to the new instllation.... My https://churchtools.gemeinsam-gemeinde.de/  works fine, but the update also have make many problems, so I have made an new installation..
Actually I have only problems with the CSV-Import from excel, but I have fount an workarround with the phpadmin
